### ISSUE TITLE
Too long opening file 

### ISSUE THREAD

**Issue by PZherdev** [ClosedXML/ClosedXML#1112]
## Read and complete the full issue template

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML**

e.g. 0.94.2

**What is the current behavior?**

I'm trying to open files thas sizes is 163

Kb, but it takes about 2-3 minutes to load it

**What is the expected behavior or new feature?**

File doesn't big it should take a more less time

**Did this work in previous versions of our tool?  Which versions?**

Yes/v0.90

**Code to reproduce problem:**
```c#
using (var workbook = new XLWorkbook(fromFilePath, XLEventTracking.Disabled)) // While executing this call
{ 

 .....

}
 
```
[File.xlsx](https://github.com/ClosedXML/ClosedXML/files/2702306/File.xlsx)

**Comment by igitur** [comment]
@Pankraty I'll need your help on this one. Looks like the recursive `Modify()` methods for settings styles caused this performance degradation.

**Comment by Pankraty** [comment]
> but it takes about 2-3 minutes to load it

You were lucky; for me, it took 8:34 at version 0.94.1 X)

Good news, the problem seems to be fixed with revision b98bfb0353ea0a6bdf7fa54567283b49b313126b: after this commit at started to open in just 14 seconds. My latest PRs (not merged) go further, lowering time to open to 8 seconds or so.

Closed with #1065.
